entrypoint/clusterreset/clusterreset_test.go

```diff
@@ -0,0 +1,109 @@
+package clusterreset
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Verifies ClusterIP Service", func() {
+		testcase.TestServiceClusterIp(true, true)
+	})
+
+	It("Verifies NodePort Service Before Reset", func() {
+		testcase.TestServiceNodePort(true, false)
+	})
+
+	It("Verifies Killall", func() {
+		testcase.TestKillall()
+	})
+
+	It("Verifies Stopped Server", func() {
+		testcase.StopServer()
+	})
+
+	It("Verifies Cluster Reset", func() {
+		testcase.ClusterReset()
+	})
+
+	It("Verifies Database Directories Deleted", func() {
```

Comment:
Update `Data` instead of `Database`

Reply:
Does this include changing the function from ```TestDeleteDatabaseDirectories``` to ```TestDeleteDataDirectories```?